docs/wiki/Wallet.md

```diff
@@ -0,0 +1,499 @@
+* **Temproary Trading Account:** dcrwallet account to temporarily store split tx outputs or change from chained swaps in
+multi-lot orders. This should only be set if 'Change Account Name' is set.
+* **RPC Username:** dcrwallet's 'username' setting for JSON-RPC.
+* **RPC Password:** dcrwallet's 'password' setting for JSON-RPC.
+* **RPC Address:** dcrwallet's address (host or host:port) (default port: 9110).
+* **TLS Certificate:** Path to the dcrwallet TLS certificate file.
+* **Wallet Name:** the name of the external wallet.
+* **JSON-RPC Username:** electrum or full node 'rpcuser' setting.
+* **JSON-RPC Password:** electrum or full node 'rpcpassword' setting.
+* **JSON-RPC Address:** electrum or full node address, <addr> or <addr>:<port> (default 'localhost').
+* **JSON-RPC Port:** electrum or full node port for RPC connections (if not set in rpcbind).
+* **Wallet File:** full path to the wallet file (empty is default_wallet).
+
+## Wallet Settings Functions
+
+* **Change Wallet Type:** Allows changing the wallet to Native (SPV), electrumRPC or full node, depending on the asset.
+* **Rescan:** Begins a rescan for all relevant transactions involving all active addresses and watched outpoints.
+Rescans can be time-consuming depending on the amount of data that must be checked, and the size of the blockchain.
+* **Recover:** Moves all wallet data to a backup folder and re-initializes the wallet.
+You will have to wait until the wallet syncs with the network, which could potentially take a long time, 
+before you can use the wallet again. 
+* **Export Wallet:** Export the wallet's private key for recovery on another Ethereum wallet. Using an externally 
+restored wallet while you have active trades running in Bison Wallet could result in failed trades and lost funds.
+It is recommended that you **do not** export your wallet unless you are an experienced user and you know what are doing.
+* **Disable Wallet:** This function will disable the wallet, it will not be connected to when you start Bison Wallet
+and cannot be used until it is re-enabled. This will also disable all token wallets that depend on this wallet.
+* **Manage Peers:** Allows adding/removing of network peers.
+* **Wallet Logs:** Opens the log file for the wallet.
+* **Rescan:** Begins a rescan for all relevant transactions involving all active addresses and watched outpoints.
+Rescans can be time-consuming depending on the amount of data that must be checked, and the size of the blockchain.
+* **Disable Wallet:** This function will disable the wallet, it will not be connected to when you start Bison Wallet
+and cannot be used until it is re-enabled. This will also disable all token wallets that depend on this wallet.
+
+# Fee Status
+
+The fee status section contains summary information around the fees associated with the selected asset.
+
+* **Network Fee Rate:** fee rate is the number of units per byte that a user pays to add their transaction 
+to the specified blockchain.
```

Comment:
For account based assets its per gas unit not byte, but maybe don't need to say.

Reply:
changed it to per byte or gas unit